tox.ini

```
@@ -1,12 +1,13 @@
 # Requires tox > 1.8
 
 [tox]
-envlist = py27-django111, py36-django{111,22}
+envlist = py27-django111, py36-django{111,22}, py39-django22
```

SHOULD: Python2.7とDjango1.11はEOLなので削除したいです。Django3.2対応は別途かな...
```suggestion
envlist =  py{36,39}-django22
```

---

Python 2.7と、Django 1.11をdropしました。